File: BusinessCardSystem.Domain/BusinessCards/BusinessCard.cs
using BizCardSystem.Domain.Abstractions;
using BizCardSystem.Domain.Enums;
using BizCardSystem.Domain.Shared;


namespace BizCardSystem.Domain.BusinessCards;

public class BusinessCard : Entity
{

    public string Name { get; set; }
    public Gender Gender { get; set; }
    public DateTime DateofBirth { get; set; }
    public string Email { get; set; }
    public string Phone { get; set; }
    public string Photo { get; set; }
    public Address Address { get; set; }
}

File: src/BizCardSystem.Presentation/ClientApp/src/app/modules/businesscards/BusinessCard.ts
import { Gender } from "../enums/Gender";
import { Address } from "./Address";

export interface BusinessCard {
    readonly id: number; 
    name: string;
    gender: Gender;
    dateofBirth: string;
    email: string;
    phone: string;
    photo: string;
    address: string;
  }